### ISSUE TITLE
Constructor annotations?

### ISSUE THREAD

**Issue by erikvanoosten** [eed3si9n/treehugger#39]
I am trying to create code like this:

    class SomeType @JsonCreator() (@JsonProperty(required = true) a: String) { ... }

How does one add an annotation to the primary constructor?

**Comment by vsapronov** [comment]
@erikvanoosten Have you figured out how to do this `@JsonCreator()` annotation?

**Comment by erikvanoosten** [comment]
Yes, we do post processing of the output. Its very ugly but works.
